cmd/observe/observe.go

```diff
@@ -179,6 +179,12 @@ programs attached to endpoints and devices. This includes:
 	observerCmd.Flags().Var(filterVar(
 		"http-status", ofilter,
 		"Show only flows which match this HTTP status code prefix (e.g. \"404\", \"5+\")"))
+	observerCmd.Flags().Var(filterVar(
+		"http-method", ofilter,
+		"Show only flows which match this HTTP method (e.g. \"get\", \"post\")"))
```

Comment:
Go has backtick to help with strings so you don't need to escape quotes

```
`Show only flows which match this HTTP method (e.g. "get", "post")"))`
```

Reply:
Done. I also updated the other uses of double quotes.